src/Microsoft.Performance.Toolkit.Engine/EngineExecutionResults.cs

```diff
@@ -41,10 +46,12 @@ public sealed class RuntimeExecutionResults
         ///     - or -
         ///     <paramref name="repository"/> is <c>null</c>.
         /// </exception>
+        /// TODO: Going to move RuntimeExecutionResults to internal and expose calls via new interface
```

Comment:
can we denote this with a `<remarks>` (or even an `ObsoleteAttribute`)

Reply:
I'm going to do this in a different PR (didn't want it to get too big :))

It will be before we snap 0.109